### PR TITLE
Revert "Create a NETCore-targeted Compilers NuGet package (#19190)"

### PR DESCRIPTION
**Pull request by agocke**
This reverts commit 2d36e41.

This change was causing the signed build to break due to publish
overwriting previously built and signed binaries.

### CODE REVIEW

**Comment by agocke** [comment]
I'm spinning up a microbuild run on this branch now

File: src/Setup/SetupStep1.proj [hunk 1]
```diff
@@ -21,6 +21,16 @@
   </ItemGroup>
 
   <Target Name="Build">
+
+    <!-- Publish CscCore and VbcCore for packaging with NuGet later -->
```
Comment (jasonmalinowski) [inline src/Setup/SetupStep1.proj]: Might want to explain why.

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
Retest this please

**Pull request merged**